modules/integration/src/test/scala/scala/cli/integration/ExportMavenTestJava.scala

```diff
@@ -0,0 +1,12 @@
+package scala.cli.integration
+
+class ExportMavenTestJava extends ExportMavenTestDefinitions with Test3Lts with MavenJava {
+  // disable running scala tests in java maven export
+  override def runExportTests: Boolean = false
+  test("pure java") {
```

Comment:
```suggestion
  if (!Properties.isWin) test("pure java") {
```
The other tests export tests apparently are only being run on mac for now. We can skip Windows here and figure it out in a future PR/issue. It passes as it should on the other platforms as is.